tcms/testcases/tests/test_views.py

```diff
@@ -228,6 +231,71 @@ def _assertTestCase(self, test_case):
         self.assertEqual(test_case.notes, self.notes)
 
 
+class TestNewCasePermission(PermissionsTestCase):
+    permission_label = 'testcases.add_testcase'
+    http_method_names = ['get', 'post']
+    url = reverse('testcases-new')
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.summary = 'summary'
+        cls.post_data = {
+            'summary': cls.summary,
+        }
+        super().setUpTestData()
+        user_should_have_perm(cls.tester, 'testcases.view_testcase')
+
+        case_status_confirmed = TestCaseStatus.get_confirmed()
+
+        case = TestCaseFactory(
+            author=cls.tester,
+            default_tester=None,
+            reviewer=cls.tester,
+            case_status=case_status_confirmed,
+        )
+        case.save()  # creates a new case entry in the database
+
+        cls.post_data.update({
```

Comment:
This makes pylint unhappy but is a false positive in this case. There is a line `# pylint: disable=objects-update-used` in TestEditCase which you can move to the top of this file, together with the other disables and it will silence pylint.

Reply:
done @atodorov